snooty/test_language_server.py

```diff
@@ -134,7 +134,7 @@ def test_text_doc_get_page_ast() -> None:
         # Change image path to be full path
         index_ast_string = (
             """<root>
-            <target ids="['guides']"></target>
+            <target domain="std" name="label" target=\"guides\"></target>
```

Comment:
Why `target=\"guides\"` output with the extra backslashes? The other fields don't include them.

Reply:
Oh, good catch. I was escaping them, which is unnecessary because it's a triple-quote string!